src/lib/MuncherSettings.js

```diff
@@ -145,10 +145,10 @@ const MuncherSettings = {
     const daeInstalled = effectModulesAvailable.daeInstalled;
     const advancedMacrosText = isNewerVersion(11, game.version) ? ` Advanced Macros${MuncherSettings.getInstalledIcon("advancedMacrosInstalled")},` : "";
     const itemMacroText = isNewerVersion(11, game.version) ? ` Item Macro${MuncherSettings.getInstalledIcon("itemMacroInstalled")}` : "";
-    const basicEffectsText = `These require DAE${MuncherSettings.getInstalledIcon("daeInstalled")}. For best automation results Midi-QOL${MuncherSettings.getInstalledIcon("midiQolInstalled")},${advancedMacrosText}${itemMacroText} Times Up${MuncherSettings.getInstalledIcon("timesUp")}, and Convenient Effects${MuncherSettings.getInstalledIcon("convenientEffectsInstalled")} are strongly recommended.`;
+    const basicEffectsText = `Requires DAE${MuncherSettings.getInstalledIcon("daeInstalled")}, Midi-QOL${MuncherSettings.getInstalledIcon("midiQolInstalled")}, ${advancedMacrosText}${itemMacroText}, Times Up${MuncherSettings.getInstalledIcon("timesUp")}, and Convenient Effects${MuncherSettings.getInstalledIcon("convenientEffectsInstalled")}.`;
     const featureEffectText = `Generate effects for a character. Some effects are always generated, some are optional (see below). ${basicEffectsText}`;
     const spellEffectText = `Generate effects for spells. These require DAE${MuncherSettings.getInstalledIcon("daeInstalled")}, Midi-QOL${MuncherSettings.getInstalledIcon("midiQolInstalled")},${advancedMacrosText}${itemMacroText} Times Up${MuncherSettings.getInstalledIcon("timesUp")}, and Convenient Effects${MuncherSettings.getInstalledIcon("convenientEffectsInstalled")} as a minimum. Also recommened: Active Auras${MuncherSettings.getInstalledIcon("activeAurasInstalled")}, Active Token Effects${MuncherSettings.getInstalledIcon("atlInstalled")}, Token Magic FX${MuncherSettings.getInstalledIcon("tokenMagicInstalled")}, Automated Animations${MuncherSettings.getInstalledIcon("autoAnimationsInstalled")} and  Warpgate${MuncherSettings.getInstalledIcon("warpgateInstalled")}.`;
-    const itemEffectText = `Generate effects for a characters equipment, doesn't include AC effects on armor, but will for things like the Ring of Protection. ${basicEffectsText}`;
+    const itemEffectText = `Generate ActiveEffects for a character's equipment. This is intended for highly automated games only. Doesn't include AC effects on armor, but will for things like the Ring of Protection. ${basicEffectsText}`;
```

Comment:
Again, the level of automation applied is dependant on if midi, or other modules are present

Reply:
It requires DAE, and recommends midi.